pkg/oci/load_balancer_spec.go

```diff
@@ -244,43 +271,62 @@ func getBackends(nodes []*v1.Node, nodePort int32) []loadbalancer.BackendDetails
 	return backends
 }
 
-func getBackendSets(svc *v1.Service, nodes []*v1.Node) map[string]loadbalancer.BackendSetDetails {
+func getBackendSets(svc *v1.Service, nodes []*v1.Node, sslCfg *SSLConfig) map[string]loadbalancer.BackendSetDetails {
 	backendSets := make(map[string]loadbalancer.BackendSetDetails)
 	for _, servicePort := range svc.Spec.Ports {
 		name := getBackendSetName(string(servicePort.Protocol), int(servicePort.Port))
-		backendSets[name] = loadbalancer.BackendSetDetails{
-			Policy:        common.String(DefaultLoadBalancerPolicy),
-			Backends:      getBackends(nodes, servicePort.NodePort),
-			HealthChecker: getHealthChecker(svc),
+		port := int(servicePort.Port)
+		var secretName string
+		if sslCfg != nil {
+			secretName = sslCfg.BackendSetSSLSecretName
+		}
+		sslConfig := getSSLConfiguration(sslCfg, secretName, port)
+		if sslConfig != nil {
```

Comment:
No need for if statement. `loadbalancer.BackendSetDetails{SslConfiguration: sslConfig}` when `sslConfig == nil` is equivalent to not specifying the property as the default value for pointers is nil.